### PR TITLE
Add geolocation support for tracking user location

### PR DESCRIPTION
**Pull request by Arasthel**
In a project I'm doing I realized that I needed to show the user location on the map. As I didn't find any way to directly do this, I coded something that should work on both native and webview versions.

### CODE REVIEW

File: library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java [hunk 1]
```diff
@@ -231,6 +231,16 @@ public void onMapClick(LatLng latLng) {
     polyline.removeFromGoogleMap();
   }
 
+  @Override
+  public void startTrackingUserLocation() {
```
Comment (nwadams) [inline library/src/main/java/com/airbnb/android/airmapview/NativeGoogleMapFragment.java]: We already have a `setMyLocationEnabled` method in the `NativeGoogleMapFragment`. We should add support for that method in AirMapView and the WebViewMapFragment (using your logic, just renaming the method calls)

**Comment by nwadams** [comment]
this looks pretty good. Can you post a screenshot of what it looks like in the web map?

**Comment by Arasthel** [comment]
It looks like [this](http://i.imgur.com/jpVt401.jpg).

As for the already implemented methods, I think everything should be fine now.

File: library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java [hunk 2]
```diff
@@ -181,7 +199,17 @@ public void setOnMarkerClickListener(OnMapMarkerClickListener listener) {
   }
 
   @Override public void setMyLocationEnabled(boolean b) {
```
Comment (nwadams) [inline library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java]: lets name this variable better. maybe `trackUserLocationEnabled

File: library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java [hunk 3]
```diff
@@ -181,7 +199,17 @@ public void setOnMarkerClickListener(OnMapMarkerClickListener listener) {
...
-    // no-op
+    this.trackUserLocation = b;
```
Comment (nwadams) [inline library/src/main/java/com/airbnb/android/airmapview/WebViewMapFragment.java]: same here for variable name.

**Comment by nwadams** [comment]
thanks for the improvement. We should have an updated package coming shortly with this included.

**Pull request merged**